docs/UserGuide.adoc

```diff
@@ -181,6 +194,7 @@ Format: `cancelappt <ENTRY_ID>`
 
 Changes the timing for an existing appointment. +
```

Comment:
Changes the timing for an existing appointment based on the given index after calling `appointment -id <REFERENCE_ID>` command.

Reply:
==== Cancels an appointment: `cancelappt` ====
Cancels the specified appointment after calling `appointments -id <REFERENCE_ID>` +
Format: `cancelappt <ENTRY_ID>`